src/Compilers/CSharp/Portable/Emitter/EditAndContinue/CSharpSymbolMatcher.cs

```diff
@@ -32,10 +32,11 @@ internal sealed class CSharpSymbolMatcher : SymbolMatcher
             EmitContext sourceContext,
             SourceAssemblySymbol otherAssembly,
             EmitContext otherContext,
-            ImmutableDictionary<ISymbolInternal, ImmutableArray<ISymbolInternal>> otherSynthesizedMembersOpt)
+            ImmutableDictionary<ISymbolInternal, ImmutableArray<ISymbolInternal>> otherSynthesizedMembersOpt,
```

Comment:
>otherSynthesizedMembersOpt [](http://example.com/codeflow?start=82&length=26)

not your change, but `otherSynthesizedMembersOpt` should probably be nullable also (and w/o `Opt` suffix)